storage/innobase/buf/buf0flu.cc

```diff
@@ -2611,25 +2611,24 @@ static
 void
 page_cleaner_adapt_lru_sleep_time(
 /*==============================*/
-	ulint*	lru_sleep_time,	/*!< in/out: desired page cleaner thread sleep
+	ulint*	lru_sleep_time)	/*!< in/out: desired page cleaner thread sleep
 				time for LRU flushes  */
-	ulint	lru_n_flushed) /*!< in: number of flushed in previous batch */
 
 {
 	ulint free_len = buf_get_total_free_list_length();
-	ulint max_free_len = srv_LRU_scan_depth * srv_buf_pool_instances;
+	ulint lru_len = UT_LIST_GET_LEN(buf_pool_from_array(0)->LRU);
```

Comment:
In 5.6 this is constant through server lifetime, right? Thus can be computed once?
In 5.7 this can be computed at startup and after each buffer pool resize